style.css

```diff
@@ -369,17 +407,157 @@ hr {
   padding: 12px 16px;
 }
 
+.nav-btn.active li a {
+  font-size: 32px;
+  color: white;
+  font-weight: 600;
+  line-height: 44px;
+  margin-bottom: 20px;
+}
+
+#preview {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  background-color: rgba(193, 199, 208, 0.9);
+  backdrop-filter: blur(5px);
+  z-index: 7;
+  padding: 10px;
+  margin: 0 auto;
+}
```

Comment:
Kindly reduce the effect of the `backdrop-filter` and make it a bit lighter so your design looks similar to the Figma design and also creates a good user experience.

Reply:
Ok will reduce that to 3px that is fine